arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/EditFeaturesWithFeatureLinkedAnnotation.storyboard

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="16097.2" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="XVk-fN-Z3w">
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="16087"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Edit Features With Feature Linked Annotation View Controller-->
+        <scene sceneID="Y18-FQ-jHG">
+            <objects>
+                <viewController id="XVk-fN-Z3w" customClass="EditFeaturesWithFeatureLinkedAnnotationViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="GJn-Lh-FXz">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ShL-2L-KLv" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                                <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                            </view>
+                        </subviews>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <constraints>
+                            <constraint firstAttribute="bottom" secondItem="ShL-2L-KLv" secondAttribute="bottom" id="S29-b1-app"/>
+                            <constraint firstAttribute="trailing" secondItem="ShL-2L-KLv" secondAttribute="trailing" id="TUw-FV-Ejk"/>
+                            <constraint firstItem="ShL-2L-KLv" firstAttribute="leading" secondItem="GJn-Lh-FXz" secondAttribute="leading" id="VLT-a8-Pd0"/>
+                            <constraint firstItem="ShL-2L-KLv" firstAttribute="top" secondItem="GJn-Lh-FXz" secondAttribute="top" id="hbe-fB-wc8"/>
```

Comment:
The top of the map view should be constrained to the top of the safe area so that it doesn't underlap the navigation bar, which is not translucent.